goty-server/src/test/kotlin/com/aleinin/goty/game/GameSearchServiceTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.aleinin.goty.game
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+
+@ExtendWith(MockitoExtension::class)
+internal class GameSearchServiceTest {
+    @Mock
+    lateinit var gameSearchClient: GameSearchClient
+
+    lateinit var gameSearchService: GameSearchService
+
+    @BeforeEach()
+    fun setup() {
+        gameSearchService = GameSearchService(gameSearchClient)
```

Comment:
You can annotate `GameSearchService` with `@InjectMocks` to avoid the `@BeforeEach`.